script/publish_metadata

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
```

Comment:
I'm not sure if having a script for updating the metadata is the right way? For now that would mean metadata would require manual execution of this script. 

Or should metadata get updated every time corresponding queries are executed via Airflow? Or as a separate step after that?